.changelog/2687.feature.1.md

```diff
@@ -0,0 +1,39 @@
+`oasis-test-runner`: Add benchmarking utilities
+
+- `oasis-test-runner` now accepts `--metrics.address` and `--metrics.interval`
+  parameters which are forwarded to `oasis-node` workers.
+- `oasis-test-runner` now signals `oasis_up` metric to Prometheus when a test
```

Comment:
Consider prefixing test-specific metrics with `oasis_test_` to differentiate them from metrics exported by oasis-node.

Reply:
I'm not sure. So each metrics has `job` which designates the sender. In this case, `oasis_up` signal could be sent by `validator-0`, `compute-0`, or `oasis-test-runner` for the test. So I would leave the common metric for this.